cumulus/parachains/runtimes/assets/asset-hub-rococo/tests/tests.rs

```diff
@@ -1355,3 +1357,118 @@ fn change_xcm_bridge_hub_ethereum_base_fee_by_governance_works() {
 		},
 	)
 }
+
+#[test]
+fn location_conversion_works() {
+	// the purpose of hardcoded values is to catch an unintended location conversion logic change.
+	struct TestCase {
+		description: &'static str,
+		location: Location,
+		expected_account_id_str: &'static str,
+	}
+
+	let test_cases = vec![
+		// DescribeTerminus
+		TestCase {
+			description: "DescribeTerminus Parent",
+			location: Location::new(1, Here),
+			expected_account_id_str: "5Dt6dpkWPwLaH4BBCKJwjiWrFVAGyYk3tLUabvyn4v7KtESG",
```

Comment:
This is the only correct `Parent` conversion test.

All others are simply copies of this one, you need to fix them. Didn't it look strange to you that `expected_account_id_str` is the same value for all conversions?

Please double-check your work (self-review) before asking for reviews to not waste reviewer time with things you could have easily caught yourself.

Reply:
My bad, sorry about that 😔
I didn't  find the documentation for Terminus and just decided to extrapolate parent tests to all the other parent locations. Though, still can't find it and the word itself is a bit confusing to me. 
Could you briefly explain what it points to? 
My best guess - it points to a chain itself.

I shouldn't have submitted this change in the evening...
Lessons learned.

Reply:
> My best guess - it points to a chain itself.

Correct. Terminus is a way to describe an "empty" list of `Junction`, a word for `Junctions::Here`.